javascript/.husky/pre-commit

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env sh
 cd javascript
 
-npm run lint
+npm run lint && npm run test
```

Comment:
I think we should run the `test` in pre-push hook. Allowing us to commit `wip`s
Thanks!